pkg/ruler/storage/wal/wal.go

```diff
@@ -677,6 +691,12 @@ func (a *appender) Commit() error {
 		buf = buf[:0]
 	}
 
+	// Notify so that reader waiting for it can read without needing to wait for next read ticker.
+	if a.notify != nil {
+		level.Warn(a.w.logger).Log("msg", "missing to notify WAL writes because notifier is not set")
```

Comment:
Minor grammatical fix

```suggestion
		level.Warn(a.w.logger).Log("msg", "not notifying about WAL writes because notifier is not set")
```